tests/unit/replication/MultipleBackendTask.js

```diff
@@ -57,12 +58,13 @@ describe('MultipleBackendTask', function test() {
                 site: 'test-site-2',
                 notificationConfigManager: {
                     getConfig: () => null
-                }
+                },
+                logger: fakeLogger,
             }),
         });
     });
 
-    describe('::_setupClients', () => {
+    describe.skip('::_setupClients', () => {
```

Comment:
Why is it being skipped? A comment with potentially mentioning a JIRA ticket would be welcome.

Reply:
🤦 sorry about that, forgot to remove the skip when i finished testing